src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -35,11 +34,12 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.0'">
+  <ItemGroup Condition="'$(TargetFramework)'=='net6.0'">
     <Compile Remove="**\CfgTest\DefaultNsAssmFixture.cs" />
     <Compile Remove="**\CfgTest\HbmOrderingFixture.cs" />
     <Compile Remove="**\DynamicProxyTests\PeVerifyFixture.cs" />
     <Compile Remove="**\Insertordering\**" />
+    <Compile Remove="**\Linq\ODataTests.cs" />
```

Comment:
I remember you mentioned some problem with .NET 6 and OData. Are you sure it's still broken with latest Microsoft.AspNetCore.OData  8.x version?

Reply:
Yes, sure. It requires `Web` sdk,